astroquery/splatalogue/load_species_table.py

```diff
@@ -66,10 +66,12 @@ def species_lookuptable(*, filename='splat-species.json', recache=False):
     # check to see if the file exists; if not, we run the
     # scraping routine
     if recache or not os.path.isfile(file_cache):
-        species = get_json_species_ids(filename)
+        species = get_json_species_ids(outfile=filename)
```

Comment:
there are some patchy test coverage around this load/build methods, and even `__main__`, so would be nice to clean up that, too (can be done separately, I've just spotted it now).

Reply:
yes, agreed - I already expanded test coverage a bit, I think?  but, some stuff is still not caught